ReactAndroid/src/main/java/com/facebook/react/views/image/ReactImageManager.java

```diff
@@ -93,6 +103,33 @@ public void setTintColor(ReactImageView view, @Nullable Integer tintColor) {
   }
 
   @Override
+  protected void addEventEmitters(final ThemedReactContext reactContext, final ReactImageView view) {
+    final EventDispatcher mEventDispatcher = reactContext.getNativeModule(UIManagerModule.class)
+        .getEventDispatcher();
+
+    view.setLoadingControllerListener(new BaseControllerListener<ImageInfo>() {
```

Comment:
I'd prefer to do sth similar to how we handle `onLayout` and only add that listener when `onLayout` is set in JS